bundles/EcommerceFrameworkBundle/VoucherService/TokenManager/Single.php

```diff
@@ -73,12 +73,13 @@ public function cleanupReservations($duration = 0, $seriesId = null)
      */
     public function prepareConfigurationView(&$viewParamsBag, $params)
     {
-        if ($this->getConfiguration()->getToken() != $this->getCodes()[0]['token']) {
+        $codes = $this->getCodes();
+        if ($codes && $this->getConfiguration()->getToken() != $codes[0]['token']) {
             $viewParamsBag['generateWarning'] = 'bundle_ecommerce_voucherservice_msg-error-overwrite-single';
             $viewParamsBag['settings']['Original Token'] = $this->getCodes()[0];
```

Comment:
`$viewParamsBag['settings']` gets overwritten on line 96 so we can get rid of it.

Reply:
@DzemalBecirevic 🏓

Reply:
@dvesh3  Sorry for not responding, I'm at home sick with the covid. I will fix this as soon as I can.

Reply:
Thank you. Get well soon 🙏